plugins/evaluation-plugin/src/com/intellij/cce/interpreter/CompletionInvokerImpl.kt

```diff
@@ -86,6 +93,30 @@ class CompletionInvokerImpl(private val project: Project,
     return com.intellij.cce.core.Lookup.fromExpectedText(expectedText, lookup.prefix(), suggestions, latency, resultFeatures, isNew)
   }
 
+  override fun callRename(expectedName: String, offset: Int): com.intellij.cce.core.Lookup {
+    LOG.info("Call rename. Type: $completionType. ${positionToString(editor!!.caretModel.offset)}")
+    //        assert(!dumbService.isDumb) { "Calling completion during indexing." }
+
+    val start = System.currentTimeMillis()
+
+    val suggestionsAsStrings = mutableSetOf<String>()
+
+    if (editor != null && editor!!.project != null) {
```

Comment:
you can assign `editor` to immutable variable before these checks, it'll avoid using so many `!!` operators here
also you can use not-null `project` field from the current class